app/styles/app.scss

```diff
@@ -2,41 +2,88 @@ $blue: #4c66b2;
 $white: white;
 $red: #ff0033;
 
-body{
+
+//==========  Mobile First Method  ==========
+
+  // Custom, iPhone Retina
+@media only screen and (min-width : 320px) {
+  .btn-danger {
+    width: 100%;
+    float: none;
+  }
+
+}
+
+  // Extra Small Devices, Phones
+@media only screen and (min-width : 480px) {
+
+}
+
+  // Small Devices, Tablets
+@media only screen and (min-width : 768px) {
+  .navbar-nav {
+    float: right;
+  }
+
+}
+
+  // Medium Devices, Desktops
+@media only screen and (min-width : 992px) {
+  .btn-danger {
+    width:180px;
```

Comment:
Properties should be ordered height, width<br>Colon after property should be followed by one space